src/richie/apps/courses/templates/courses/cms/organization_detail.html

```diff
@@ -1,6 +1,13 @@
 {% extends "richie/fullwidth.html" %}
 {% load cms_tags extra_tags i18n static thumbnail %}
 
+{% block meta-keywords %}{{ current_page.get_title|lower }}{% endblock %}
```

Comment:
why do you set keywords with a title and only for this page?
If we have nothing for keywords for the moment, we could skip this meta for the moment and see later how we can include here the list of linked categories?

Reply:
Removed